--- include/linux/if_vlan.h	2021-11-22 14:12:02.845505939 -0600
+++ include/linux/if_vlan_jumbo.h	2021-11-22 14:28:18.987946414 -0600
@@ -22,8 +22,8 @@
 /*
  * According to 802.3ac, the packet can be 4 bytes longer. --Klika Jan
  */
-#define VLAN_ETH_DATA_LEN	1500	/* Max. octets in payload	 */
-#define VLAN_ETH_FRAME_LEN	1518	/* Max. octets in frame sans FCS */
+#define VLAN_ETH_DATA_LEN	9000	/* Max. octets in payload	 */
+#define VLAN_ETH_FRAME_LEN	9018	/* Max. octets in frame sans FCS */
 
 #define VLAN_MAX_DEPTH	8		/* Max. number of nested VLAN tags parsed */
