Index: jessie-stage3/rootfs/etc/lightdm/lightdm-gtk-greeter.conf
===================================================================
--- jessie-stage3.orig/rootfs/etc/lightdm/lightdm-gtk-greeter.conf
+++ jessie-stage3/rootfs/etc/lightdm/lightdm-gtk-greeter.conf
@@ -16,7 +16,7 @@
 # screensaver-timeout = Timeout (in seconds) until the screen blanks when the greeter is called as lockscreen
 # 
 [greeter]
-background=/usr/share/images/desktop-base/login-background.svg
+background=#ffffff
 theme-name=Adwaita
 #icon-theme-name=
 #font-name=
