Index: jessie-stage2/rootfs/etc/network/interfaces
===================================================================
--- jessie-stage2.orig/rootfs/etc/network/interfaces
+++ jessie-stage2/rootfs/etc/network/interfaces
@@ -1,4 +1,20 @@
-auto lo
+# interfaces(5) file used by ifup(8) and ifdown(8)
+
+# Please note that this file is written to be used with dhcpcd
+# For static IP, consult /etc/dhcpcd.conf and 'man dhcpcd.conf'
 
+# Include files from /etc/network/interfaces.d:
+source-directory /etc/network/interfaces.d
+
+auto lo
 iface lo inet loopback
-iface eth0 inet dhcp
+
+iface eth0 inet manual
+
+allow-hotplug wlan0
+iface wlan0 inet manual
+    wpa-conf /etc/wpa_supplicant/wpa_supplicant.conf
+
+allow-hotplug wlan1
+iface wlan1 inet manual
+    wpa-conf /etc/wpa_supplicant/wpa_supplicant.conf
