--- dream.pro.org	2020-09-04 22:51:51.579926191 +0200
+++ dream.pro	2020-09-04 22:52:57.609434707 +0200
@@ -70,9 +70,6 @@
       exists(/opt/local/include/speex/speex_preprocess.h) {
         CONFIG += speexdsp
       }
-      exists(/opt/local/include/hamlib/rig.h) {
-        CONFIG += hamlib
-      }
       contains(QT_VERSION, ^4\\.7.*) {
 	QT += phonon opengl svg
         DEFINES -= QWT_NO_SVG
@@ -138,12 +135,6 @@
       packagesExist(sndfile) {
         CONFIG += sndfile
       }
-      packagesExist(hamlib) {
-        CONFIG += hamlib
-      }
-      packagesExist(gpsd) {
-        CONFIG += gps
-      }
       packagesExist(pcap) {
         CONFIG += pcap
       }
@@ -159,14 +150,6 @@
       exists(/usr/local/include/sndfile.h) {
         CONFIG += sndfile
       }
-      exists(/usr/include/hamlib/rig.h) | \
-      exists(/usr/local/include/hamlib/rig.h) {
-          CONFIG += hamlib
-      }
-      exists(/usr/include/gps.h) | \
-      exists(/usr/local/include/gps.h) {
-        CONFIG += gps
-      }
       exists(/usr/include/pcap.h) | \
       exists(/usr/local/include/pcap.h) {
         CONFIG += pcap
@@ -194,9 +177,6 @@
     exists($$OUT_PWD/include/speex/speex_preprocess.h) {
       CONFIG += speexdsp
     }
-    exists($$OUT_PWD/include/hamlib/rig.h) {
-      CONFIG += hamlib
-    }
     exists($$OUT_PWD/include/pcap.h) {
       CONFIG += pcap
     }
@@ -225,7 +205,7 @@
          LIBS += -lz
     }
 }
-exists($$OUT_PWD/include/neaacdec.h) {
+exists(/usr/include/neaacdec.h) {
      DEFINES += HAVE_LIBFAAD \
      USE_FAAD2_LIBRARY
      LIBS += -lfaad_drm
@@ -257,11 +237,6 @@
      win32:LIBS += libspeexdsp.lib
      message("with libspeexdsp")
 }
-gps {
-     DEFINES += HAVE_LIBGPS
-     unix:LIBS += -lgps
-     message("with gps")
-}
 pcap {
      DEFINES += HAVE_LIBPCAP
      unix:LIBS += -lpcap
@@ -269,24 +244,6 @@
      win32-g++:LIBS += -lwpcap -lpacket
      message("with pcap")
 }
-hamlib {
-     DEFINES += HAVE_LIBHAMLIB
-     macx:LIBS += -framework IOKit
-     unix:LIBS += -lhamlib
-     win32:LIBS += libhamlib-2.lib
-     HEADERS += src/util/Hamlib.h
-     SOURCES += src/util/Hamlib.cpp
-     qt {
-       HEADERS += src/util-QT/Rig.h
-       SOURCES += src/util-QT/Rig.cpp
-     }
-     gui {
-         HEADERS += src/GUI-QT/RigDlg.h
-         SOURCES += src/GUI-QT/RigDlg.cpp
-         FORMS += RigDlg.ui
-     }
-     message("with hamlib")
-}
 qwt {
     DEFINES += QWT_NO_SVG
     macx {
