Index: jessie-stage4/rootfs/etc/lightdm/lightdm.conf
===================================================================
--- jessie-stage4.orig/rootfs/etc/lightdm/lightdm.conf
+++ jessie-stage4/rootfs/etc/lightdm/lightdm.conf
@@ -114,7 +114,7 @@
 #session-setup-script=
 #session-cleanup-script=
 #autologin-guest=false
-#autologin-user=
+autologin-user=pi
 #autologin-user-timeout=0
 #autologin-in-background=false
 #autologin-session=UNIMPLEMENTED
