Index: jessie-stage1/rootfs/lib/udev/rules.d/75-persistent-net-generator.rules
===================================================================
--- jessie-stage1.orig/rootfs/lib/udev/rules.d/75-persistent-net-generator.rules
+++ jessie-stage1/rootfs/lib/udev/rules.d/75-persistent-net-generator.rules
@@ -22,7 +22,7 @@ IMPORT{cmdline}="net.ifnames"
 ENV{net.ifnames}=="1",			GOTO="persistent_net_generator_end"
 
 # device name whitelist
-KERNEL!="eth*|ath*|wlan*[0-9]|msh*|ra*|sta*|ctc*|lcs*|hsi*", \
+KERNEL!="ath*|msh*|ra*|sta*|ctc*|lcs*|hsi*", \
 					GOTO="persistent_net_generator_end"
 
 # ignore Xen virtual interfaces
