--- stage2.orig/rootfs/etc/init.d/rng-tools	2020-09-20 17:48:03.967820377 +0100
+++ stage2/rootfs/etc/init.d/rng-tools	2020-09-21 04:30:07.379806813 +0100
@@ -6,9 +6,11 @@
 #
 ### BEGIN INIT INFO
 # Provides:		rng-tools
-# Required-Start:	$remote_fs $syslog
-# Required-Stop:	$remote_fs $syslog
-# Default-Start:	2 3 4 5
+# Required-Start:
+# Required-Stop:	unmoutroot
+# Should-Stop:
+# X-Start-Before:    	fake-hwclock
+# Default-Start:     	S
 # Default-Stop:		0 1 6
 ### END INIT INFO
 #
@@ -19,7 +21,7 @@
 DAEMON=/usr/sbin/rngd
 NAME=rngd
 DESC="Hardware RNG entropy gatherer daemon"
-PIDFILE=/var/run/rngd.pid
+PIDFILE=/run/rngd.pid
 
 DEVICELIST="hwrng hw_random hwrandom intel_rng i810_rng"
 
